clientlibs/js/src/ApiService/ApiService.ts

```diff
@@ -302,23 +302,18 @@ export default class ApiService {
     });
   }
 
-  public async getAllFeatureFlags(): Promise<IFeatureFlag[]> {
-    const response = await this.sendRequest<IFeatureFlag[], never>({
+  public async getAllFeatureFlags(): Promise<string[]> {
+    const requestBody: UpGradeClientRequests.IGetAllExperimentConditionsRequestBody = {
```

Comment:
Should have `IGetAllFeatureFlagRequestBody` which internally could be same

Reply:
code updated